Makefile

```diff
@@ -74,10 +76,19 @@ endif
 	@printf "The engine has been cleaned.\n"
 
 version: variables
+ifneq ($(OVERWRITE_MOD_VERSION),"True")
+	@printf "Mod version $(MOD_VERSION) will remain unchanged.\n"
+else 	
 	@awk '{sub("Version:.*$$","Version: $(VERSION)"); print $0}' $(MANIFEST_PATH) > $(MANIFEST_PATH).tmp && \
 	awk '{sub("/[^/]*: User$$", "/$(VERSION): User"); print $0}' $(MANIFEST_PATH).tmp > $(MANIFEST_PATH) && \
 	rm $(MANIFEST_PATH).tmp
-	@printf "Version changed to $(VERSION).\n"
+ifeq ("$(MOD_VERSION)", "$(VERSION)")
+# Avoiding "Packaging mod version x as x" after the first buildpackage.sh has overwritten the MOD_VERSION.
+	@printf "Setting mod version to $(VERSION)\n"
+else
+	@printf "Packaging mod version $(MOD_VERSION) as $(VERSION)\n"
```

Comment:
This message IMO is unnecessarily confusing.
By example:
```
paul@ubuntu-vm:~/src/OpenRAModSDK$ make version VERSION=1
Packaging mod version {DEV_VERSION} as 1
paul@ubuntu-vm:~/src/OpenRAModSDK$ make version VERSION=2
Packaging mod version 1 as 2
paul@ubuntu-vm:~/src/OpenRAModSDK$ make version VERSION=3
Packaging mod version 2 as 3
paul@ubuntu-vm:~/src/OpenRAModSDK$ make version VERSION=4
```

Note that none of these commands are run from the packaging scripts :wink:

IMO it would be better to simply print "Setting mod version to $(VERSION)\n" in all cases.